curriculum/challenges/english/01-responsive-web-design/css-piano/part-014.md

```diff
@@ -0,0 +1,88 @@
+---
+id: 612e825311abd03445acf69e
+title: Part 14
+challengeType: 0
+dashedName: part-14
+---
+
+# --description--
+
+Add that same class to the third, fifth, sixth, and seventh `.key` elements.
```

Comment:
**Note**: An issue I have with these kinds of lessons is, what if a Camper stopped (took a few days break) on the previous lesson, and comes back to this one? It is slightly confusing being told to _do the same_ ...

_Small nitpick, I know_ 🤷‍♂️